### ISSUE TITLE
fzf keybinding doesn't work

### ISSUE THREAD

**Issue by timurb** [rockandska/fzf-obc#35]
With plain fzf integration I can hit Ctrl-R and search history with fzf.
With fzf-obc the standard "reverse-i-search" is used.

This probably can be fixed by simply sourcing  `/usr/local/opt/fzf/shell/key-bindings.bash` as it was previously done in `~/.fzf.bash`

**Comment by rockandska** [comment]
Indeed, because i never planed to replace the other fzf default capabilities who works great.
I just forgot that fzf install script doesn't had keybindings and do its updaterc by default and should add this in install instructions if a user install fzf for the first time.
